code/controllers/subsystem/traumas.dm

```diff
@@ -20,35 +22,106 @@ SUBSYSTEM_DEF(traumas)
+						/obj/item/device/soulstone/,
+						/obj/structure/destructible/clockwork/, /obj/item/clockwork/, /obj/item/clothing/suit/armor/clockwork, 
+						/obj/item/clothing/glasses/judicial_visor, /obj/effect/clockwork/sigil/, /obj/item/stack/tile/brass, 
+						/obj/machinery/door/airlock/clockwork/,
+						/obj/item/clothing/suit/wizrobe, /obj/item/clothing/head/wizard, /obj/item/spellbook, /obj/item/staff, 
+						/obj/item/clothing/suit/space/hardsuit/shielded/wizard, /obj/item/clothing/suit/space/hardsuit/wizard,
+						/obj/item/gun/magic/staff/, /obj/item/gun/magic/wand/,
+						/obj/item/nullrod, /obj/item/clothing/under/rank/chaplain)),
+						
+						"aliens"   = typecacheof(list(/obj/item/clothing/mask/facehugger, /obj/item/organ/body_egg/alien_embryo,
+						/obj/structure/alien/, /obj/item/toy/toy_xeno,
+						/obj/item/clothing/suit/armor/abductor/, /obj/item/device/abductor/, /obj/item/gun/energy/alien, 
+						/obj/item/abductor_baton, /obj/item/device/radio/headset/abductor, /obj/item/scalpel/alien, /obj/item/hemostat/alien,
+						/obj/item/retractor/alien, /obj/item/circular_saw/alien, /obj/item/surgicaldrill/alien, /obj/item/cautery/alien,
+						/obj/item/clothing/head/helmet/abductor, /obj/structure/bed/abductor, /obj/structure/table_frame/abductor, 
+						/obj/structure/table/abductor, /obj/structure/table/optable/abductor, /obj/structure/closet/abductor, /obj/item/organ/heart/gland,
+						/obj/machinery/abductor, /obj/item/crowbar/abductor, /obj/item/screwdriver/abductor, /obj/item/weldingtool/abductor, 
+						/obj/item/wirecutters/abductor, /obj/item/wrench/abductor, /obj/item/stack/sheet/mineral/abductor))
 					   )
-	phobia_turfs = list("space" = typecacheof(list(/turf/open/space, /turf/open/floor/holofloor/space, /turf/open/floor/fakespace)))
 
-	phobia_species = list("lizards"   = typecacheof(list(/datum/species/lizard)),
-						  "skeletons" = typecacheof(list(/datum/species/skeleton, /datum/species/plasmaman))
+	phobia_turfs = list("space" = typecacheof(list(/turf/open/space, /turf/open/floor/holofloor/space, /turf/open/floor/fakespace)),
+						"the supernatural" = typecacheof(list(/turf/open/floor/clockwork, /turf/closed/wall/clockwork, 
+						/turf/open/floor/plasteel/cult, /turf/closed/wall/mineral/cult)),
+						"aliens" = typecacheof(list(/turf/open/floor/plating/abductor, /turf/open/floor/plating/abductor2, 
+						/turf/open/floor/mineral/abductor, /turf/closed/wall/mineral/abductor))
+						)
+
+	phobia_species = list("lizards"   = typecacheof(list(/datum/species/lizard/)),
+						  "skeletons" = typecacheof(list(/datum/species/skeleton, /datum/species/plasmaman)),
+						  "robots"   = typecacheof(list(/datum/species/android)),
+						  "the supernatural" = typecacheof(list(/datum/species/golem/clockwork, /datum/species/golem/runic)),
+						  "aliens" = typecacheof(list(/datum/species/abductor, /datum/species/jelly/, /datum/species/pod, 
```

Comment:
Trailing / on the jelly.

Reply:
Intended, it's so I don't have to write down each subspecies or make any additions if more jelly subspecies get added

Reply:
You don't need the trailing / for that, it works that way by default

Reply:
Oh it does? Very well then